backstop/tests/header.html

```diff
@@ -641,7 +641,7 @@ <h2>
             <!-- Project Button -->
             <nav aria-label="breadcrumbs">
               <span class="iui-button-split-menu iui-header-split-button">
-                <div>
+                <div class="iui-button-group-item">
```

Comment:
Do we really need here button group? It is split button and we know how its gonna be (no possible customizations). So maybe instead we could just have those few needed rules in that button css and do not depend on button group mixin? 
This class looks out of context now.

Reply:
Also noticed, in split menu, we are importing button group mixin twice :O

Reply:
we could remove button-group here, but then we should also do that in the base split-button

Reply:
Yeah, so thats what I am asking. Cant we do same with split button as you did with paginator?

Reply:
With paginator it was easy because we don't need any of the custom button-group logic.

For split-button, we need the border radius handling so I would have to duplicate that part. That's not really the point of this PR, so I would rather do it in a separate PR.

Reply:
Well, this PR changes the split button, which is not nice. It can be done with separate PR, but Id prefer this not to be released without split buttons fix.

Reply:
This class is removed so it's no longer a concern